www/src/app/views/InboxContainer.js

```diff
@@ -223,6 +219,22 @@ export default class InboxContainer extends Component {
         handleResolve={this.handleResolve}
         handleArchive={this.handleArchive}
       />
+      </section>
     );
   }
 }
+
+const mapStateToProps = (state) => {
+  return {
+    author: state.user.author,
+    updates: state.updates.inbox
```

Comment:
`inbox` is just one possible status an update can have.  The other are: `event`, `todo`, `done`, `goal`, `struggle`, `achievement`, `archive` and `removed`. I think this line should simply read:

    updates: state.updates